packages/restapi/src/lib/chat/helpers/validator.ts

```diff
@@ -22,20 +22,12 @@ export const createGroupRequestValidator = (
         throw new Error(`groupDescription cannot be more than 150 characters`);
     }
 
-    if (members == null) {
-        throw new Error(`members cannot be null`);
-    }
-
     for (let i = 0; i < members.length; i++) {
         if (members[i] && !isValidETHAddress(members[i])) {
             throw new Error(`Invalid member address!`);
         }
     }
 
-    if (admins == null) {
```

Comment:
We have still have this check to ensure admin is empty array rather than null

Reply:
@Aman035  I check the create group request type, members or admins cannot be null because of the type. Check below export interface ChatCreateGroupType extends EnvOptionsType {
  account?: string | null;
  signer?: SignerType | null;
  groupName: string;
  groupDescription: string;
  members: Array<string>;
  groupImage: string | null;
  admins: Array<string>;
  isPublic: boolean;
  contractAddressNFT?: string;
  numberOfNFTs?: number;
  contractAddressERC20?: string;
  numberOfERC20?: number;
  pgpPrivateKey?: string | null;
  meta?: string;
  groupType? : string | null,
  scheduleAt ?: Date | null;
  scheduleEnd?: Date | null;
}